storefront/components/CTA/CTA.tsx

```diff
@@ -23,13 +23,25 @@ const CTA = (props: Props) => {
 export default CTA;
 
 const StyledButton = styled.button`
-  width: 100%;
+  width: 50%;
   padding: 16px 20px;
   border: none;
-  background: ${(props) => props.theme.colors.secondary};
+  margin-left:25%;
+  margin-right:25%;
```

Comment:
This wasn't asked into the related issue... and not a good practice to let a component define his own margins. Always better to let the Container that wraps him specify margins & paddings. CSS Flexbox & Grid are also geat for this job.